lib/task/import/csvAccessionImportTask.class.php

```diff
@@ -238,6 +243,58 @@ public function execute($arguments = array(), $options = array())
             }
           }
 
+          // Add alternative identifiers
+          $identifiers = $self->rowStatusVars['alternativeIdentifiers'];
+          $identifierNotes = $self->rowStatusVars['alternativeIdentifierNotes'];
+
+          if (!empty($identifiers) || !empty($identifierNotes))
+          {
+            $identifierTypes = $self->rowStatusVars['alternativeIdentifierTypes'];
+
+            for ($index = 0; $index < max(count($identifiers), count($identifierNotes)); $index++)
+            {
+              $identifier = (empty($identifiers[$index])) ? null : $identifiers[$index];
+
+              if (!empty($identifier) || !empty($identifierNotes[$index]))
+              {
+                $otherName = new QubitOtherName;
+                $otherName->object = $self->object;
+                $otherName->name = $identifier;
```

Comment:
If the alt Identifier column is blank for a row but altIdNotes is populated, will it create othername records with blank names?

Reply:
It will, luckily! The archival spec makes the identifiers optional (so I guess you could put in a note that an alternative identifier of some sort is needed).

Reply:
CAAIS spec allows for the actual identifier to be optional.